### PR TITLE
Secondary auditors workbook

### PR DESCRIPTION
**Pull request by danswick**
Addresses #1280

### CODE REVIEW

File: backend/schemas/source/base/FederalProgramNames.json
Comment (danswick) [inline backend/schemas/source/base/FederalProgramNames.json:1]: I don't think this file should have changed. It probably got caught up in a merge issue.

File: backend/schemas/output/excel/xlsx/additional-ueis-template.xlsx
Comment (danswick) [inline backend/schemas/output/excel/xlsx/additional-ueis-template.xlsx:1]: I ran the schema `make` command after catching up to main and it updated all of the other workbooks, which may be expected? They only appear to have changed by a few bytes apiece.

**Comment by danswick** [comment]
@jadudm am I missing a process for adding new excel templates to the repo? After some stumbling around, I'm running into some issues because my new excel template is gitignored.

File: backend/schemas/output/excel/xlsx/additional-ueis-template.xlsx
Comment (jadudm) [reply backend/schemas/output/excel/xlsx/additional-ueis-template.xlsx:1]: Make always remakes all the files. We're currently keeping the workbooks in the tree, and this is gonig to be a side-effect of that. Normally, build products would not be in the tree.

When time permits, I'd be game to have a way to tag/release workbooks via actions. For now...

**Review by sambodeme — APPROVED** [review]

**Pull request merged**